boilerplates/react-union-boilerplate-liferay-basic/src/apps/SampleApp/index.js

```diff
@@ -22,7 +23,9 @@ ready(() => {
 	render(Root);
 
 	if (window.Liferay) {
-		window.Liferay.on('startNavigate', justUnmountComponentAtNode);
+		// NOTE: cannot be shortened because justUnmountComponentAtNode needs to default the first arg
+		// TODO: refactor this to use the new `attachListeners` API
```

Comment:
I'll remove these comments and use `justUnmountComponentAtNode` like before. This is for another PR.